.github/workflows/versioning.yml

```diff
@@ -9,15 +9,12 @@ jobs:
     steps:
       - name: Git checkout
         uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      # Updates the v* branch to reflect the content of the main branch
-      # This allows versioning `plugins.json`, providing consumers use the
-      # branch-specific deploy URL like
-      #   v*--netlify-plugins.netlify.app/plugins.json
+      # Updates the list-v* tag to version `plugins.json`.
+      # Consumers must use the tag-specific deploy URL:
+      #   list-v*--netlify-plugins.netlify.app/plugins.json
       - name: Update branch
         run: |
-          git branch -C main v1
-          git push --force -u origin v1
+          git tag --force list-v1
+          git push --force --tags
```

Comment:
Both `--force` are required here for updates.

Reply:
Can we update the step name to `Create tag?